### PR TITLE
jormungandr: priority to journey with bike_in_pt while eliminating similar journey

### PR DESCRIPTION
**Pull request by kadhikari**
Concerned ticket: http://jira.canaltp.fr/browse/NAVITIAII-2221

Hand-tested only (OK whether tagged `bike_in_pt` or not)

### CODE REVIEW

File: source/jormungandr/jormungandr/scenarios/new_default.py [hunk 1]
```diff
@@ -800,6 +800,7 @@ def call_kraken(self, request_type, request, instance, krakens_call):
         resp = []
         logger = logging.getLogger(__name__)
         futures = []
+        self.nb_kraken_calls = 0
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/scenarios/new_default.py]: why did you add this ?

**Comment by kadhikari** [comment]
jenkins test this please

File: source/jormungandr/jormungandr/scenarios/new_default.py (hunk 1 above)
Comment (kadhikari) [reply source/jormungandr/jormungandr/scenarios/new_default.py]: This initializes the counter of kraken call and is not concerned to bike_in_pt.

**Comment by xlqian** [comment]
Jenkins, test this please!

**Review by TeXitoi — APPROVED** [review]

**Review by TeXitoi — APPROVED** [review]
